### PR TITLE
Initialise get all user

### PR DESCRIPTION
**Pull request by SimonLab**
First version of the plugin. Expose getAllUsers function which returns all the users (active and not)
@JMurphyWeb @mk4111 any comments?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database ce100_test;' -U postgres
```
Comment (mk4111) [inline .travis.yml:15]: I know this is just for travis, but do we want to set a specific database that we want to test/configure all our plugins in?

**Review by mk4111 — APPROVED** [review]
I've had a quick look and I think it lgtm!!

File: .travis.yml (hunk 1 above)
Comment (SimonLab) [reply .travis.yml:15]: Yes we should decide on a common name for the database across our plugins but at the moment it's not a blocker and more an enhance feature.

**Comment by SimonLab** [comment]
@mk4111 thanks for the review :+1:

**Pull request merged**